draft-ietf-quic-transport.md

```diff
@@ -2128,18 +2128,21 @@ clients to further narrow applicability or reuse.
 
 ## Path Validation {#migrate-validate}
 
-Path validation is used during connection migration (see {{migration}}) by the
-migrating endpoint to verify reachability of a peer from a new local address.
-In path validation, endpoints test reachability between a specific local address
-and a specific peer address, where an address is the two-tuple of IP address and
-port.
+Path validation is used by both peers during connection migration
+(see {{migration}}) to verify reachability after a change of address.
+In path validation, endpoints test reachability between a specific local
+address and a specific peer address, where an address is the two-tuple of
+IP address and port.
 
-Path validation tests that packets sent on a path to a peer are
-received by that peer. Path validation is used to ensure that packets received
-from a migrating peer do not carry a spoofed source address.
+Path validation tests that packets sent on a path to a peer are received by that
+peer. Path validation ensures that packets received from a migrating peer do
+not carry a spoofed source address.
 
 Path validation does not validate that a peer can send in the return direction.
-The peer performs independent validation of the return path.
+Acknowledgments cannot be used for path validation as they contain insufficient
+entropy and might be spoofed. No method is provided to establish return
+reachability, as endpoints independently determine reachability on each
+direction of a path.
```

Comment:
I found `No method is provided to establish return reachability` confusing. How about:

```suggestion
entropy and might be spoofed. Endpoints independently determine reachability
on each direction of a path, and therefore return reachability can only be
established by the peer.
```